### PR TITLE
build on CircleCI

### PR DESCRIPTION
**Pull request by zoff99**
small tweaks
check actual exit code

---
This change is [Reviewable](https://reviewable.io/reviews/toktok/dockerfiles/12)

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+machine:
+  timezone:
+    Europe/Berlin
+  services:
+    - docker
+  environment:
+    DOCKERFLAGS: --rm=false
+dependencies:
+  pre:
+  - sudo apt-get update
```
Comment (iphydf) [inline circle.yml]: None of these seem necessary.

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    - echo "dummy"
+  override:
+    - cat /etc/apt/sources.list
+    - ls -al /etc/apt/sources.list.d
+    - docker info
+    - make
+    - make -C ghc-android build-arm > $CIRCLE_ARTIFACTS/make_out.txt 2>&1 ; if [ $? -eq 0 ]; then touch ~/ok.txt ; fi ; touch ~/ready.txt :
+        background: true
+
+    - ready_="0";
```
Comment (iphydf) [inline circle.yml]: Lots of repetition. Perhaps put this code in a separate script and call it from here?

File: circle.yml (hunk 1 above)
Comment (zoff99) [inline circle.yml]: DOCKERFLAGS: --rm=false
was added by you :-)

File: circle.yml (hunk 2 above)
Comment (zoff99) [inline circle.yml]: no

File: circle.yml (hunk 1 above)
Comment (zoff99) [inline circle.yml]: apt-get update can never hurt

**Comment by iphydf** [comment]
:lgtm_strong:

---

Review status: 0 of 4 files reviewed at latest revision, 1 unresolved discussion.

---

*[circle.yml, line 29 at r1](https://reviewable.io:443/reviews/toktok/dockerfiles/12#-KmCAdfAyuIzSz0TrEMy-r1-29:-L1bd61OJBRO4mCmILI9:b-igqyzg) ([raw file](https://github.com/toktok/dockerfiles/blob/c56ef03a301b675b34d72179026a2ce07a6f0f44/circle.yml#L29)):*
<details><summary><i>Previously, zoff99 (Zoff) wrote…</i></summary><blockquote>

no
</blockquote></details>

\*Shrug\*.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/dockerfiles/12#-:-L1bd4LX0J5Iv9QQl0kQ:b-jbsf8z)*

**Pull request merged**